src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -155,7 +155,7 @@
   attachmentFileInfo: {
     fontSize: dimen.normalizeDimen(12),
     fontFamily: fonts.inter[400],
-    color: COLORS.gray
+    color: COLORS.gray500
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the attachmentFileInfo has been changed. Ensure that this new color provides sufficient contrast against its background for readability.

```diff
-    color: COLORS.gray
+    color: COLORS.gray500
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->